### PR TITLE
The page is a better copy-paste template now

### PR DESCRIPTION
**Pull request by WebReflection**
As mentioned in t[his review](https://github.com/WebReflection/hyperHTML/commit/ae5c40f990e49bb9e746fe5e1355b9aaa08e5d2f#commitcomment-21199748), the incremental demo could be semantically better, as it is in here.

### CODE REVIEW

**Comment by WebReflection** [comment]
@marcoscaceres let me know if this works, I've also spot a little bug thanks to this update 🎉

File: test/incremental.html [hunk 1]
```diff
@@ -12,25 +12,28 @@
...
+  function update() {
+    render`
+      Incremental updates for ${items.length} items
+      <form onsubmit="${e => confirm('Are you sure ?')}">
+      <input type="submit">${
+        items.map((item, i) => hyperHTML.wire(item)`
+          <input name="${'input-' + i}">
+        `)
+      }</form>
+    `
```
Comment (marcoscaceres) [inline test/incremental.html]: Nit: missing ";"

**Comment by marcoscaceres** [comment]
Yeah, this looks good!

File: test/incremental.html (hunk 1 above)
Comment (WebReflection) [inline test/incremental.html]: Geeeeez, how dare me ?

**Pull request merged**